3D/Web/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Object Viewer</title>
+    </head>
+    <body>
+        <canvas id="bg"></canvas>
+        <script type="module" src="./src/main.js"></script>
```

Comment:
Could we make this a react component that can be used in the main frontend? Like a 
`<ArheoViewer siteID={selectedSiteID}>` 

And that component could fetch the SiteSection based on the user's selected Site.

Reply:
Possibly, I have a member of my team working on replicating this functionality in Fiber.

Reply:
Great then!